lib/ibanity/api/sandbox/financial_institution_transaction.ex

```diff
@@ -299,6 +302,9 @@ defmodule Ibanity.Sandbox.FinancialInstitutionTransaction do
       mandate_id: {~w(attributes mandateId), :string},
       creditor_id: {~w(attributes creditorId), :string},
       additional_information: {~w(attributes additionalInformation), :string},
+      fee: {~w(attributes fee), :string},
```

Comment:
Should this be a float?

Reply:
it should use the same type as 'amount' indeed. Decimal may be safer though

Reply:
🤦 auwch that explains the string in Ponto. I've gone and replaced all floats for decimals (as is the case in core), alright?

Reply:
Sticking to floats for now, I think I've got them all now